### PR TITLE
Add issue triage config

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

https://devbots.xyz/documentation/needs-triage/

Requested by @tchaton

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [n/a] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by justusschock — APPROVED** [review]

File: .devbots/needs-triage.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# adds a triage 5 minutes after creation.
+# triagers should replace this label with any appropriate labels, then, the cc-bot will cc interested parties.
+enabled: true
+label: "needs triage"
+delay: 5
+# issues only have assignees, milestones, or these labels when a core contributor created them.
+# no need to add the triage label in that case
+skip_when_has_assignee: true
+skip_when_has_milestone: true
+skip_when_has_labels: ["priority: 0", "priority: 2", "priority: 3"]
```
Comment (justusschock) [inline .devbots/needs-triage.yml:10]: can't we add any label here that is not part of the templates? (like all the module-specific labels)

Comment (carmocca) [reply .devbots/needs-triage.yml:10]: We can add wildcard `*` but the issue templates add `bug`, `feature`, `refactor` labels by default so we cannot use it

Comment (justusschock) [reply .devbots/needs-triage.yml:10]: Yes, but all the others we can, right? So whenever something except `bug`, `feature`, `refactor`, `docs` is present.

Comment (carmocca) [reply .devbots/needs-triage.yml:10]: Alternatively we can either list here all other labels
Or remove the default additions from the issue templates and have triagers label them

Comment (carmocca) [reply .devbots/needs-triage.yml:10]: Going with the 2nd option as it would be annoying to open a PR every time a label is added

**Review by awaelchli — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

**Comment by Borda** [comment]
Could you please clarify why removing existing labels?

**Comment by carmocca** [comment]
@Borda see the comment in the code and the discussions here: https://github.com/PyTorchLightning/pytorch-lightning/pull/11880#discussion_r806076300

**Review by Borda — APPROVED** [review]

**Pull request merged**